src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -6998,6 +6998,60 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 tmp = ASR::make_LogicalConstant_t(al, loc, is_lower,
                         ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
                 return;
+            } else if (attr_name == "isalpha") {
+                /*
+                * Specification:
+                * Return True if all characters in the string are alphabetic (letters), and False otherwise.
+                */
+                bool is_alpha = true;
+                for (auto &i : s_var) {
+                    if (!std::isalpha(i)) {
+                        is_alpha = false;
+                        break;
+                    }
+                }
+                tmp = ASR::make_LogicalConstant_t(al, loc, is_alpha,
+                        ASRUtils::TYPE(ASR::make_Logical_t(al, loc, 4)));
+                return;
+            } else if (attr_name == "istitle") {
+                /*
+                * Specification: Return True if the string is in title case, where the first
+                * letter of each word is capitalized and the rest are lowercase. Return False
+                * if the string is empty or does not meet the title case criteria.
```

Comment:
I would mention here which specific characters the `istitile()` ignores/skips. I would also share two examples here about some (small) sentences (couple of words with spaces and special character) and what `istitle()` would return for them.

```
... does not meet the title case criteria.
```

Or I would share about the `title case critera` (possibly using example(s).

Reply:
Added few lines